### DISCUSSION TITLE
EFB Warning

### DISCUSSION THREAD

**Discussion by TheHi-Champion** [magknight/787aviatorsTracker#628] [Ideas]
Some people tend to have a lighting issue, it would be nice to have the EFB warning to put the Visual effects to maximum for best lighting quality.

HDR may be difficult to find for some people so telling them, this would help them a lot.

**Comment by aeroniemi** [comment]
check whether there's a *hdr enabled* dataref
might need to make it a runOnce item (save state to config file), although equally might not be worth it
might need to/be worth adding the multiple message code to the efb, maybe classed

**Comment by Simleybob** [comment]
Not much point in doing this now, since xp12 will force it on/there won't be an option to have it off
